src/Runner/MediaInfoCommandRunner.php

```diff
@@ -63,10 +62,10 @@ public function __construct($filePath, array $arguments = null, $processBuilder
      */
     public function run()
     {
-        $this->processBuilder->add($this->filePath);
+        $lc_ctype = setlocale(LC_CTYPE, 0);
+        $this->processBuilder->add($this->filePath)->addEnvironmentVariables(array('LANG' => $lc_ctype));
```

Comment:
@alangregory could you replace ```addEnvironmentVariables(array('LANG' => $lc_ctype)``` by ```setEnv('LANG', $lc_type)``` https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Process/ProcessBuilder.php#L142